applications/crossbar/doc/storage.md

```diff
@@ -0,0 +1,103 @@
+### Storage
+
+#### About Storage
+
+#### Schema
+
+Key | Description | Type | Default | Required
+--- | ----------- | ---- | ------- | --------
+`attachments` |   | `#/definitions/storage.attachments` |   | `false`
+`connections` |   | `#/definitions/storage.connections` |   | `false`
+`plan` |   | `#/definitions/storage.plan` |   | `false`
+
+
+#### Remove
```

Comment:
Please don't push "empty" docs like this: it makes tools & humans believe storage is documented when actually only ref/storage.md was copied